cloudformation_templates/aws_route53_digital_services_store.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
This file doesn't contain anything specific to Digital Services Store - should/could it have a more generic name in case it might be re-used in future for other domains?  Or are there things here that we really only want to use for DSS?